tsconfig.json

```diff
@@ -34,6 +34,7 @@
   "include": [
     "src/composables/types.js",
     "src/composables/use-event-listener-outside.js",
-    "src/composables/use-active-audio.js"
+    "src/composables/use-active-audio.js",
+    "src/utils/key-codes.js",
```

Comment:
Might as well include this even though it's not related. Doesn't hurt anything and will make the `key-codes.js` module easier to use anyway.